tests/integration/_experimental/test_quality_standard.py

```diff
@@ -104,6 +111,11 @@ def test__download_quality_standard_result(
         quality_standard_df,
         download_quality_standard_result(dataset_name, [model_name], [metric_group_name]),
     )
+    assert_frame_equal(
+        download_quality_standard_result(dataset_name, [model_name, model_name2]),
+        download_quality_standard_result(dataset_name, [model_name, model_name2], intersect_results=False),
+        check_like=True,
+    )
```

Comment:
if we don't upload partial results for model2, i think this test is not that usefully

Reply:
ok, let me just separate the tests